### PR TITLE
refactor(website, last updated): update demo layout

### PR DESCRIPTION
**Pull request by GermainBergeron**
### Proposed Changes

Added JsDoc on LastUpdated props
Updated LastUpdated demo layout

image

### Potential Breaking Changes

None

### Acceptance Criteria

-   [ ] The proposed changes are covered by unit tests
-   [ ] The potential breaking changes are clearly identified
-   [ ] [README.md](https://github.com/coveo/plasma/blob/master/README.md) is adjusted to reflect the proposed changes (if relevant)

### CODE REVIEW

File: packages/website/src/examples/LastUpdated/LastUpdatedSpecifiedTime.example.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import {LastUpdated} from '@coveord/plasma-react';
+import * as React from 'react';
+
+const ONE_HOUR = 60 * 60 * 1000;
+
+export default () => {
+    const OneHourAgo = new Date(new Date().getTime() - ONE_HOUR);
+    return <LastUpdated time={OneHourAgo} label="Dernière modification à" />;
```
Comment (toofff) [inline packages/website/src/examples/LastUpdated/LastUpdatedSpecifiedTime.example.tsx:8]: Hoo French, is it normal?

File: packages/react/src/components/lastUpdated/LastUpdatedConnected.tsx [hunk 2]
```diff
@@ -22,4 +22,7 @@ const mapDispatchToProps = (
     onDestroy: () => dispatch(removeLastUpdated(ownProps.id)),
 });
 
+/**
+ * @deprecated use LastUpdated instead
+ */
 export const LastUpdatedConnected = connect(mapStateToProps, mapDispatchToProps, ReduxUtils.mergeProps)(LastUpdated);
```
Comment (toofff) [inline packages/react/src/components/lastUpdated/LastUpdatedConnected.tsx:28]: I just looked in the Admin project we don't use it anymore.

**Review by toofff — APPROVED** [review]

**Review by aiheon — APPROVED** [review]

File: packages/website/src/examples/LastUpdated/LastUpdatedSpecifiedTime.example.tsx (hunk 1 above)
Comment (GermainBergeron) [reply packages/website/src/examples/LastUpdated/LastUpdatedSpecifiedTime.example.tsx:8]: Well it's just an example to show that you can customize the label

File: packages/react/src/components/lastUpdated/LastUpdatedConnected.tsx (hunk 2 above)
Comment (GermainBergeron) [reply packages/react/src/components/lastUpdated/LastUpdatedConnected.tsx:28]: Yep! That's why I marked it as deprecated.

We'll do one pass soon to remove a bunch of deprecated components in one go

**Pull request merged**